Makefile

```diff
@@ -80,13 +80,20 @@ generate: manifests $(CONTROLLER_GEN) $(GOIMPORTS) $(MOCKGEN)
 test: $(GINKGO) $(GOTESTFMT)
 	# run ginkgo unit tests. These will be ported to golang native tests over a period of time.
 	@"$(HACK_DIR)/test.sh" ./internal/controller/etcdcopybackupstask/... \
-	./internal/controller/predicate/... \
 	./internal/controller/secret/... \
 	./internal/controller/utils/... \
 	./internal/mapper/... \
-	./internal/metrics/...
+	./internal/metrics/... \
+	./internal/health/...
 	# run the golang native unit tests.
-	@TEST_COV="true" "$(HACK_DIR)/test-go.sh" ./api/... ./internal/controller/etcd/... ./internal/controller/compaction/... ./internal/component/... ./internal/utils/... ./internal/webhook/...
+	@TEST_COV="true" "$(HACK_DIR)/test-go.sh" ./api/... \
+	./internal/controller/etcd/... \
+	./internal/controller/compaction/... \
+	./internal/component/... \
+	./internal/errors/... \
+	./internal/store/... \
+	./internal/utils/... \
+	./internal/webhook/...
```

Comment:
These warnings are due to usage of `gotestfmt`.
Even though i have configured it run with `-hide  empty-packages` (see https://github.com/GoTestTools/gotestfmt?tab=readme-ov-file#how-do-i-make-the-output-less-verbose)  but i guess it does not work with empty nested packages. You can ignore it for now. I will replace this test output formatter soon.

Reply:
Also it is interesting you still see `/sentinel` package (cached). Maybe you can clear your cache and run it again.

Reply:
It seems like we need to clear the cache each time.

Reply:
> These warnings are due to usage of `gotestfmt`. Even though i have configured it run with `-hide empty-packages` (see https://github.com/GoTestTools/gotestfmt?tab=readme-ov-file#how-do-i-make-the-output-less-verbose) but i guess it does not work with empty nested packages. You can ignore it for now. I will replace this test output formatter soon.

It works without the `-cover` option 😕

Reply:
that is true but then you lose the coverage details. I know it a small irritant at this time but do you want to remove coverage because of some extra lines in the test output?

Reply:
I agree, let's ignore the warnings. I'm fine with it.